tests/test_crypto.py

```diff
@@ -4454,3 +4484,20 @@ def test_does_not_contain(self):
             [get_elliptic_curve(self.curve_factory.another_curve_name)]
         )
         assert curve not in curves
+
+
+def utf8_pem():
+    """
+    Read a certificate, create a x509 object and print his subject
+    """
+    x509 = OpenSSL.crypto.load_certificate(
+        OpenSSL.crypto.FILETYPE_PEM, cert_utf8
+    )
+    print(x509.get_subject())
+
+
+def test_utf8_pem():
+    """
+    Test utf8_pemt() function and raises an error in case of UTF8 fixture
+    """
+    assert utf8_pem() is None
```

Comment:
You don't need a seperate `utf8_pem` function, and please do not print the certificate. Also, please assert something more specific about the result.